### DISCUSSION TITLE
Map rotation mode where it aligns to real world

### DISCUSSION THREAD

**Discussion by crackwitz** [streetcomplete/StreetComplete#4893] [Ideas]
Currently, the app can orient the map in two ways:
- point the map's north *forward* (device) by pressing the wind rose
- point map relative to latest segment of track

When I have a look around in a place, trying to get my bearings, I'd like the map to align to the real world, i.e. the map's north pointing physical north.

I'd like that to be a mode you can switch into by pressing the wind rose, if the view is currently pointing north (or the wind rose was pressed just now without manual rotations since)

**Comment by Helium314** [comment]
The closest to what you want would be the old compass mode, see #3335 (switch from compass to movementr direction), #3377 (request to get compass mode back), #3954 (abandoned PR for compass mode).

The compass mode was removed because the compass direction often is rather loosely related to the "real world".
